CommandLineClient/src/main/java/com/dslplatform/compiler/client/Utils.java

```diff
@@ -258,10 +259,18 @@ private static void logCommand(final Context context, final ProcessBuilder build
 	}
 
 	public static boolean testCommand(final Context context, final String command, final String contains) {
-		return testCommand(context, command, contains, new ArrayList<String>());
+		return testCommand(context, command, contains, Collections.<String>emptyList());
+	}
+
+	public static boolean testCommand(final Context context, final String command, final String contains, final Charset charset) {
```

Comment:
I preferring adding overloads only if they are commonly used. Not just because

Reply:
Undided.